src/storage/storage_common.h

```diff
@@ -157,6 +157,7 @@ struct log_lsa
 /*
  * Sector
  */
+#define BOOT_VOLUME_MINSECTS 64
```

Comment:
remove BOOT_VOLUME_MINSECTS and BOOT_VOLUME_MINPAGES.
remove unused function boot_max_pages_for_new_temp_volume.

create extern function in disk manager to return DISK_MIN_VOLUME_SECTS.